### PR TITLE
testing: don't print the output of marimo unless there was an error

### PR DESCRIPTION
**Pull request by superlopuh**

### CODE REVIEW

**Comment by superlopuh** [comment]
```
MLIR is installed, running tests.
/home/runner/work/xdsl/xdsl/llvm-project/build/bin//mlir-opt
Running docs/marimo/mlir/onnx_demo.py
All marimo mlir tests passed successfully.
```

File: Makefile [hunk 1]
```diff
@@ -54,7 +54,11 @@ tests-toy: filecheck-toy pytest-toy
 tests-marimo:
 	@for file in docs/marimo/*.py; do \
 		echo "Running $$file"; \
-		python3 "$$file" || exit 1; \
+		error_message=$$(python3 "$$file" 2>&1) || { \
+			echo "Error running $$file"; \
+			echo "$$error_message"; \
+			exit 1; \
```
Comment (webmiche) [inline Makefile:60]: Can we make it s.t. it does not fail on the first failing one, but runs all of them and then reports the errors or something?

**Review by webmiche — APPROVED** [review]
LGTM and works on my machine. Just a minor comment :+1:

File: Makefile (hunk 1 above)
Comment (superlopuh) [reply Makefile:60]: https://github.com/xdslproject/xdsl/issues/2841

**Pull request merged**